ansible-advancedlinux/azuredeploy.parameters.json

```diff
@@ -1,51 +1,33 @@
 {
-  "$schema": "http://schema.management.azure.com/schemas/2015-05-01/deploymentParameters.json#",
+  "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentParameters.json#",
```

Comment:
This should be http://schema.management.azure.com/schemas/2015-05-01/deploymentParameters.json#